h/presenters/annotation_json.py

```diff
@@ -17,18 +17,21 @@ class AnnotationJSONPresenter(AnnotationBasePresenter):
 
     """Present an annotation in the JSON format returned by API requests."""
 
-    def __init__(self, annotation_resource):
+    def __init__(self, annotation_resource, formatters=[]):
```

Comment:
I know you mention this in your PR message. Could we perhaps change this to `formatters=None` and a couple of extra lines below? Your code is fine, but seeing `foo=[]` in a parameter list is always going to cause people to have to stop and think, and thinking _hurts_ 🤕.